mailbox/store/src/test/java/org/apache/james/mailbox/store/mail/model/ListMailboxAssert.java

```diff
@@ -27,48 +27,28 @@
 import org.apache.james.mailbox.store.mail.model.impl.SimpleMailbox;
 
 import com.google.common.base.Function;
+import com.google.common.base.Objects;
 import com.google.common.collect.FluentIterable;
 
 public class ListMailboxAssert {
```

Comment:
We had "MailboxAssert" to compare Mailbox by its properties but in our cases, we have to compare the list of JPAMailbox and SimpleMailbox.
The method "isEqualToComparingOnlyGivenFields" is better to replace for MailboxAssert but I cannot do it because the Mailbox and SimpleMailbox do not have the same property names even. One has "id" and another has "mailboxId" so I think we need to refactor it. Do we have any other ways?